src/SprykerEco/Zed/Heidelpay/Persistence/Propel/Schema/spy_heidelpay.schema.xml

```diff
@@ -72,6 +72,45 @@
         <id-method-parameter value="spy_payment_heidelpay_credit_card_registration_pk_seq"/>
     </table>
 
+    <table name="spy_payment_heidelpay_direct_debit_registration">
+        <column name="id_direct_debit_registration" required="true" type="INTEGER" autoIncrement="true" primaryKey="true"/>
+        <column name="fk_customer_address" required="false" type="INTEGER" />
+
+        <column name="registration_unique_id" required="true" type="VARCHAR" />
+        <column name="transaction_id" required="true" type="VARCHAR" />
+
+        <column name="account_holder" required="false" type="VARCHAR" />
+        <column name="account_bank_name" required="false" type="VARCHAR" />
+        <column name="account_number" required="false" type="VARCHAR" />
+        <column name="account_country" required="false" type="VARCHAR" />
+        <column name="account_bic" required="false" type="VARCHAR" />
+        <column name="account_iban" required="false" type="VARCHAR" />
```

Comment:
I guess we need length restriction for such fields

Reply:
Do you think such validations should be done on DB level?